packages/trie/benchmarks/index.ts

```diff
@@ -1,45 +1,50 @@
-import Benchmark = require('benchmark')
+import Benchmark from 'benchmark'
 import { runTrie } from './random'
 import { iterTest } from './checkpointing'
+import { MemoryDB, LevelDB } from '../dist'
 
 const suite = new Benchmark.Suite()
 
-// random.ts
-// Test ID is defined as: `pair_count`-`era_size`-`key_size`-`value_type`
-// where value_type = symmetric ? 'mir' : 'ran'
-// The standard secure-trie test is `1k-9-32-ran`
-// https://eth.wiki/en/fundamentals/benchmarks#results-1
-suite
-  .add('1k-3-32-ran', async () => {
-    await runTrie(3, false)
-  })
-  .add('1k-5-32-ran', async () => {
-    await runTrie(5, false)
-  })
-  .add('1k-9-32-ran', async () => {
-    await runTrie(9, false)
-  })
-  .add('1k-1k-32-ran', async () => {
-    await runTrie(1000, false)
-  })
-  .add('1k-1k-32-mir', async () => {
-    await runTrie(1000, true)
-  })
+for (const [name, DB] of Object.entries({ MemoryDB, LevelDB })) {
```

Comment:
hey thanks for the great PR, continued work and interest!

I'm not sure if it's worth doubling the number of benchmarks run here on every PR. What we could do instead is a one time profile difference between the two and record the results in the readme, and save the script so we could add in the future like lmdb comparison.

Reply:
Sure a BENCHMARK.md sounds good too, you can link to it from the readme so people can discover it. There is already a trie-benchmarks GitHub action so my intention was to not need to run the comparison every time.